### ISSUE TITLE
[PROJECT] Developing Support Ops 

### ISSUE THREAD

**Issue by BenJam** [opencollective/opencollective#5906]
> As a member of the support team I want to fix issues for users

> As a member of the engineering team I want to reduce the amount of time I spend supporting users

> As an organisation we want to ensure we do not treat support as an excuse for excessive tech debt

## Support tasks

Below are a selection of common support ops tasks:

* #5771
* #5651
* #5374
* #5000
* #4848
* #4841
* #4805
* #4714
* #4684
* #4648
* #4627
* #3981

to which I would like to add

* #5907

**Comment by BenJam** [comment]
It helps to have one issue that we can reference in conversation around sprints (and general direction/strategy) so I'm reopening this.

**Comment by poohlaga** [comment]
Adding a link to the support issue tracker https://github.com/orgs/opencollective/projects/5/views/30
and a link to the work from design to update the help/support page https://github.com/opencollective/opencollective/issues/4800

**Comment by BenJam** [comment]
To summarise the conversation held on Friday, there are _many_ problems that need to be addressed here. Some are process-driven and others are product driven. 

I'm inclined to leave the process-driven work to @alinamanko when she takes on her new role, but for the moment I'll describe all the problems here as best I can:

### Addressing the top of funnel

> Lots of our user support requests are RTFMs

Last year we prioritised improvements to our help pages https://opencollective.com/help

We incorporated direct links to sections of our documentation. The design also incorporated a search which was discounted due to limitations with GitBook #5022. We'll take a look again at that and, if it continues to be a blocker, we may look at migrating our docs to a better provider. 

- [ ] look again at implementing docs search as designed #5022

I cannot go without saying that, as part of this workstream we _must_ review our platform documentation in the round. Our docs are inconsistent and arguably platform-centric (that is, they concern features) rather than user centric (that is, they concern user's behaviour) and that might be worth reviewing. We may need to hire in someone as a consultant or full time member of staff to do this. I'll leave the decision to @alinamanko as someone heading up the support and ops in the near future. 

- [ ] review documentation at docs.opencollective.com 

### Increasing the efficiency of the support team

Firstly, it is obvious that we are abusing Freshdesk as a catch-all for email that does not belong in a single-person's inbox. Invoices, contracts, vendor agreements and _much worse_ are routed through our freshdesk and this causes added strain on the team. If we're going to scale our support capacity we need to streamline the information that's coming into it. 

- [ ] review emails routed to FreshDesk and remove all non-support notices 

secondly it's not obvious how tickets are routed, and who's responsibility support is. Much of the support that's routed to @poohlaga is based on an understandably quick triage process based on the user, the collective and who their host is. We should make it _patently clear_ when we expect the host to pick up a support request, and when it's the responsibility of the platform provider. 

- [ ] establish platform support policy for hosts

@shannondwray is currently triaging tickets in Freshdesk, adding the users profile and other information before assigning it to someone else in the team (see [this workflow](https://miro.com/app/board/uXjVPL9o7t8=/?share_link_id=52229209537)). We previously looked at attaching pertinent information (username, profile link, collective they're a member of, most recent transaction etc) to a support request #5023. We'll look again at this from a product perspective.

- [ ] look again at #5023 and decide on information that would be useful to accompany a freshdesk ticket automatically

Finally the development support for ops and user support is limited to @SudharakaP. It's obvious from speaking that Suds has very little time for prospective (as opposed to reactive) work in this area. It's important that we allow time for Suds to work on automating the tasks that he frequently, manually, addresses such as resetting a users access. Thsi may require the rest of the team offloading some support elsewhere. Finally @SudharakaP and I will review the outstanding support issues in GitHub and prioritise based on where he would free the most of his time 

- [ ] review support issues with @SudharakaP and prioritise based on time dedicated at the moment.

**Comment by shannondwray** [comment]
From my knowledge Alina is taking a step back from Support and this is an area I’ve been asked to step up in and take some ownership rather than following Alinas lead. 
- @carolineawoolard and I have gone into depths surrounding my role and part of this is handing off Documentation to someone who is more skilled, has more capacity. We would need to touch base with Caroline to see what the timeline looks like for this and what I can do to prepare for the new Documentation person. 

Here is the Docs issue about finding a new documentation provider: https://github.com/opencollective/opencollective/issues/2595

I am going to visualise out all of the automations that happen through freshdesk so we can see what is coming through freshdesk and where it is being sent etc 

I will also try and create a clear distinction between what is platform support and what is host support - putting things in the middle so we can clearer ensure we debate and clearer define the scope

**Comment by BenJam** [comment]
Thanks @shannondwray and apologies if I got that wrong. @carolineawoolard and I are already picking up the discussion on hiring someone to dig into docs.

**Comment by BenJam** [comment]
So @SudharakaP and I went through the support and ops section of the project board https://github.com/orgs/opencollective/projects/5/views/30 and removed anything that's not a support engineer scale. 

We prioritised issues that will _reduce the burden on @SudharakaP's time_ for the moment, namely:

- #5771
- #5651
- #4714

These are our immediate focus. Issues that are 'support and operations' but not a support engineer scale (that is, can be worked by @SudharakaP alone) are moved to the 'support and operations product area for prioritisation. 

FYI @shannondwray: for the moment I have also asked that @SudharakaP take a selfish approach to handling tickets *rejecting anything that can be done without direct database access*.

**Comment by shannondwray** [comment]
Amazing!! Please send things back my way so I can ensure I know how to do them and can reflect this in the support how to/visualisation flow

**Comment by piamancini** [comment]
> From my knowledge Alina is taking a step back from Support and this is an area I’ve been asked to step up in and take some ownership rather than following Alinas lead.

@shannondwray you are correct that she is taking a step back from frontline support but she will be taking a higher level view in terms of support processes and technical improvements. 

This means: 
- working in categorizing the type of support that is platform related to the one that should be sent to the host, 
- looking into technical improvements such as a support chat or AI service to filter incoming support tickets, 
- working with engineering to use freshdesk's API to connect to platform. 
- mentoring support / ops people from the different host. 

You will be working together with Alina in this tasks I believe - @carolineawoolard please add here if you have any additional comments.

**Comment by alinamanko** [comment]
@BenJam thank you so much for this summary and apologies for missing the capacity to join the conversation in person. 

Just to add to what @shannondwray commented and to offer an extension to what @piamancini said and what I commented during our support ops meeting the other day. I will be indeed focusing more on the support and ops aspects from the platform perspective and working with the frontline support to improve the functionality we are now starting to focus more now with the incredible work the support focused team is already doing.  

My stepping back in an immediate term is due to some personal circumstances and the need to regroup in the hope of returning with much more capacity to give when I move to the new role in March.

**Comment by BenJam** [comment]
- [ ] some comments to resolve on https://github.com/opencollective/opencollective-api/pull/8470

**Comment by BenJam** [comment]
@shannondwray @natehn could we run through the list above and update our current position on improvements for support and operations before the end o the year?

**Comment by shannondwray** [comment]
Hello! We are reviewing all of the outstanding issues and will put forward a proposal from the Support team based on our prioritization
